Source/CarthageKit/GitHub.swift

```diff
@@ -97,7 +97,7 @@ public struct GitHubRepository: Equatable {
 }
 
 public func ==(lhs: GitHubRepository, rhs: GitHubRepository) -> Bool {
-	return lhs.owner == rhs.owner && lhs.name == rhs.name
+	return lhs.owner.lowercaseString == rhs.owner.lowercaseString && lhs.name.lowercaseString == rhs.name.lowercaseString
```

Comment:
`String.caseInsensitiveCompare` expresses the intent much more clearly. And who knows if we’ll allow ÜTF8 usernames someday.